app/com/linkedin/drelephant/analysis/AnalyticJob.java

```diff
@@ -304,6 +312,72 @@ public AppResult getAnalysis() throws Exception {
     // Retrieve information from job configuration like scheduler information and store them into result.
     InfoExtractor.loadInfo(result, data);
 
+    String project_name = data.getConf().getProperty("azkaban.flow.projectname");
```

Comment:
Make your code is modular and pluggable. Do not integrate it only for Azkaban. If people want to build this DAG for some other scheduler then they should be able to implement it easily.

See if you can move all the azkaban specific changes to ```AzkabanScheduler.java```